gradle/root/release.gradle

```diff
@@ -69,251 +18,37 @@ notes {
-    def envVar = System.env.TRAVIS_PULL_REQUEST
-    envVar != null && envVar != '' && envVar != 'false'
-}
-
-configurations {
-    previousSrc
-    previousPom
-}
-
-dependencies {
-    //TODO I don't like the fact that the previous version is taken from the release notes document
-    //get it from the version.properties! If there is no previous version in version properties, add it during the version bump
-    previousSrc "org.mockito:mockito-core:${project.notes.getPreviousVersion()}:sources@jar"
-    previousPom "org.mockito:mockito-core:${project.notes.getPreviousVersion()}@pom"
-}
-
-task("comparePublications", type: PublicationsComparatorTask) {
-    dependsOn publishToMavenLocal, sourcesJar
-
-    compareBinaries({ configurations.previousSrc.singleFile }, { sourcesJar.archivePath })
-    comparePoms({ configurations.previousPom.singleFile.text }, { generatePomFileForMainJarPublication.destination.text })
-}
-
-//TODO if System.env.TRAVIS_BRANCH is empty, just pick up current branch from Git
-
-/*
-Release process should *not* run concurrently.
- */
-task("releaseNeeded") {
-    ext.needed = true
-    dependsOn comparePublications
-    doLast {
-        def branch = System.env.TRAVIS_BRANCH
-        def skipEnvVariable = System.env.SKIP_RELEASE
-        def commitMessage = System.env.TRAVIS_COMMIT_MESSAGE
-        def skippedByCommitMessage = commitMessage?.contains(skipReleaseCommitMessage)
-
-        if (skippedByCommitMessage || skipEnvVariable) {
-            logger.lifecycle("Release is skipped on request")
-            ext.needed = false //we really don't want the release
-        } else if (releaseTest) {
-            logger.lifecycle("Release will be initiated for testing purposes")
-            ext.needed = true //testing purposes, dry run
-        } else if (!isPullRequest() && isReleasableBranch(branch) && !comparePublications.publicationsEqual) {
-            logger.lifecycle("All criteria are met, the release will be initiated")
-            ext.needed = true //standard, continuous delivery scenario
-        } else {
-            logger.lifecycle("Criteria are not met, the release will NOT be initiated")
-            ext.needed = false
-        }
-
-        logger.lifecycle("Release need evaluation - needed: ${ext.needed}, " +
-                "releasable branch: ${isReleasableBranch(branch)}, pull request: ${isPullRequest()}, dry run / test release: $releaseTest, " +
-                "publications equal: $comparePublications.publicationsEqual, " +
-                "skip env variable: $skipEnvVariable, skipped by message: $skippedByCommitMessage")
-    }
-}
 
-task printReleaseHelp {
-    doLast {
-        logger.lifecycle troubleshootingInfo
-    }
-    onlyIf { releaseNeeded.needed }
-}
+    git_genericUser = "Mockito Release Tools"
+    git_genericEmail = "<mockito.release.tools@gmail.com>"
+    git_releasableBranchRegex = "master|release/.+"  // matches 'master', 'release/2.x', 'release/3.x', etc.
```

Comment:
Maybe `master` should be commented out to do not accidentally generate release when merged into master?